rio/src/screen/mouse.rs

```diff
@@ -53,6 +54,18 @@ impl Default for Mouse {
     }
 }
 
+impl Mouse {
+    pub fn new(multiplier: f64) -> Self {
+        Self {
+            multiplier,
+            ..Default::default()
+        }
+    }
+    pub fn set_multiplier(&mut self, multiplier: f64) {
```

Comment:
This case we can use inline,
suggestion:

```rust
#[inline]
pub fn set_multiplier(&mut self, multiplier: f64) {
```

Reply:
edited. 38271fcee8cfa52a897150708224820cc9f19a09